v2/futures/client.go

```diff
@@ -354,6 +355,11 @@ func (c *Client) callAPI(ctx context.Context, r *request, opts ...RequestOption)
 		if e != nil {
 			c.debug("failed to unmarshal json: %s", e)
 		}
+
+		if apiErr.Code == 0 {
+			fmt.Printf("binance futures api code is 0, resp: %#v, resp body: %s, status code: %d\n", res, string(data), res.StatusCode)
+		}
+
```

Comment:
Do not print message in a base client, use `c.debug` instead. However, I don't think this change is necessary.